libs/providers/flagd-web/src/lib/flagd-web-provider.ts

```diff
@@ -129,13 +129,13 @@ export class FlagdWebProvider implements Provider {
           }
         },
         () => {
-          reject();
```

Comment:
@beeme1mr This was the main problem. I added this reject unconditionally no matter what retry we were on. This was the main issue.

Instead, I've moved an error event down below, only after we've exhausted all the retries.

Reply:
I've also tested this in the browser and it works as expected.